build/commands/migrate_settings.py

```diff
@@ -18,55 +18,31 @@
 
 class MigrateTestSettings(Command):
     """
-    Class to copy all test_settings.py files listed to settings.py
+    Class to copy test_credentials.ini to credentials.ini
     """
 
-    description = 'Overwrite the settings.py files with test_settings.py'
+    description = 'Overwrite the credentials.py files with test_credentials.py'
     user_options = []
 
     def initialize_options(self):
-        """ Initialise empty list """
-        # pylint:disable=attribute-defined-outside-init
-        self.test_settings_paths = []
+        pass
```

Comment:
Is this still needed?

Reply:
Yeah, it seems that if this function isn't overriden at all it throws a `RuntimeError`